src/program/services/libraries/symlink.py

```diff
@@ -159,24 +159,29 @@ def process_shows(directory: Path, item_type: str, is_anime: bool = False) -> Ge
             for episode in os.listdir(directory / show / season):
                 if os.path.splitext(episode)[1][1:] not in ALLOWED_VIDEO_EXTENSIONS:
                     continue
-                if not (episode_number := re.search(r"s\d+e(\d+)", episode, re.IGNORECASE)):
+                episode_numbers = re.findall(r"s\d+e(\d+)(?:-e(\d+))?", episode, re.IGNORECASE)
```

Comment:
Now that I think of it, please use RTN/parsett for this.

Reply:
use parsett, it'll be faster to process.

heres an example

```py
from PTT import parse_title

episodes: list[int] = parse_title(episode).episodes
```